### PR TITLE
migrations: add create incident table migration

### PR DESCRIPTION
**Pull request by isacikgoz**
Example PR for adding a migration :)

### CODE REVIEW

**Comment by crspeller** [comment]
I guess it's not necessary to follow the exact same migrations as we had before. 

What level of granularity should we be aiming for in these migrations? I would think they should be logical units for each feature, not versions like we had before.

File: server/sqlstore/migrations/mysql/000002_create_IR_incident.down.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS IR_Incident;
```
Comment (lieut-data) [inline server/sqlstore/migrations/mysql/000002_create_IR_incident.down.sql:1]: (Commenting on file, not this line) Should we keep the filenames lowercase for simplicity?

**Review by lieut-data — APPROVED** [review]
Thanks, @isacikgoz! If we don't map 1:1 with the "old" migrations, will it be tricky to decide how to migrate folks who upgrade, say, across ESRs? Or are we relying on the idempotency of the `.up.sql` scripts to guarantee this?

Also wondering if we should investigate a way to prove that we end up at the same state as the existing migrations after we complete this effort.

**Comment by isacikgoz** [comment]
> I guess it's not necessary to follow the exact same migrations as we had before.
> 
> What level of granularity should we be aiming for in these migrations? I would think they should be logical units for each feature, not versions like we had before.

Sorry I was off, so replying this a bit late :)

The granularity is totally up to you but it's not needed to follow versions here. We can use a table (entity) based granularity like we did for server.

**Review by crspeller — APPROVED** [review]

**Comment by lieut-data** [comment]
Thanks @isacikgoz! We'll merge this and use it as a base from which to complete these morph migrations.

**Pull request merged**